modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -445,10 +446,10 @@ def _launch_tasks(cls, splits: list, **partition_kwargs) -> Tuple[list, list, li
         Parameters
         ----------
         splits : list
-            list of tuples with partitions data, which defines
-            parser task (start/end read bytes and etc.)
+            `list` of tuples with partitions data, which defines
```

Comment:
```suggestion
            list of tuples with partitions data, which defines
```
I don't think we need to monotype-highlight this word here, numpydoc says "use backticks as references to variables"